rollup.config.js

```diff
@@ -26,8 +18,24 @@ module.exports = [
     terser(),
   ],
   output: {
-    file: path.resolve(BASE_PATH, 'dist/rulesets/oas/functions', `${fn}.js`),
+    file: path.resolve(BASE_PATH, folder, `${fn}.js`),
     format: 'cjs',
     exports: 'named'
   },
 }));
+
+module.exports = processor('dist/rulesets/oas/functions', [
+  'oasOp2xxResponse',
+  'oasOpFormDataConsumeCheck',
+  'oasOpIdUnique',
+  'oasOpParams',
+  'oasOpSecurityDefined',
+  'oasPathParam',
+  'refSiblings',
+])
+.concat(processor('dist/rulesets/oas2/functions', [
+  // Add here the oas2 specific functions
```

Comment:
@P0lip It took me quite some time to understand this part of the packaging process and, although this change doesn't add any value, I thought this might be of some help to the future contributors.

However, if you want me to get rid of this change, I'll happily do it, though.

Reply:
Nah, it's okay to leave it. It does help others to understand what's going on here.